metricbeat/tests/system/test_zookeeper.py

```diff
@@ -71,7 +71,6 @@ def test_output(self):
         self.assertEqual(len(output), 1)
         evt = output[0]
 
-        self.assertItemsEqual(self.de_dot(ZK_FIELDS), evt.keys())
```

Comment:
I see you remove this line. What were the errors you got? Perhaps there is an eays way to fix it instead of removing it.

Reply:
Error in Travis was pointing directly to this line which was pointing to a `service` field (`service.name` to be precise) that wasn't found. As far as I saw, this field comes from https://github.com/elastic/beats/blob/master/metricbeat/tests/system/metricbeat.py#L9 with this value
```python 
COMMON_FIELDS = ["@timestamp", "beat", "metricset.name", "metricset.host",
                 "metricset.module", "metricset.rtt", "host.name", "event"]
```

A way to fix this is to use a customized `COMMON_FIELDS` which doesn't seem very ideal. But in any case I'm not sure if it should contain a `service.name` or not

Reply:
I think what should work here is:

```
self.assertItemsEqual(self.de_dot(ZK_FIELDS + ["sevice"]), evt.keys())
```

Reply:
Yeah, that's customized way but I'm a bit afraid that it could bite us in the future (by getting outdated somehow).

Anyways, test passes with that error but now I get a `Key 'service.version' found in event is not documented` which is true but I don't understand why CI hasn't detected it in master :confused: